rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/110_synonyms_invalid.yml

```diff
@@ -313,3 +313,158 @@ setup:
       indices.stats: { index: test_index }
 
   - length: { indices: 0 }
+
+---
+"Load index with non existent synonyms set":
```

Comment:
Very nice test, I like how you tried to create synonym set after that

Reply:
Thanks! I see this as a way to document the way out of the error situation for the previous behaviour